test/clj/quil/snippet.clj

```diff
@@ -1,10 +1,12 @@
 (ns quil.snippet
-  (:require [quil.util :refer [clj-compilation?]]
-            [quil.core :as q]
-            [quil.util :refer [no-fn]]
+  (:require [quil.core :as q]
             [quil.snippets.all-snippets :as as]
-            [clojure.test :as t :refer [is]]
-            [clojure.string :as cstr]
+            [clojure.edn :as edn]
+            [clojure.java.io :as io]
+            [clojure.test :as t]
+            [clojure.java.shell :as sh]
+            [clojure.string :as string]
+            [etaoin.api :as e]
```

Comment:
Can we use something longer than `e` etaoin? Because `e/go` is really generic function used below and it's hard to understand where it comes from. I'd prefer having full `etaoin` to make keep it explicit.

Reply:
done